module/metrics/network.go

```diff
@@ -245,6 +246,15 @@ func NewNetworkCollector(logger zerolog.Logger, opts ...NetworkCollectorOpt) *Ne
 		}, []string{LabelNodeRole, LabelMessage, LabelChannel, LabelRateLimitReason},
 	)
 
+	nc.violationReportSkippedCount = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: namespaceNetwork,
+			Subsystem: subsystemSecurity,
+			Name:      nc.prefix + "slashing_violation_reports_skipped_count",
+			Help:      "number of slashing violations consumer violations that were not reported for misbehavior when the identity of the sender not known",
```

Comment:
```suggestion
			Help:      "number of slashing violations consumer violations that were not reported for misbehavior because the identity of the sender not known",
```